### PR TITLE
add provisioning

### PR DESCRIPTION
**Pull request by akorosov**

### CODE REVIEW

**Comment by mortenwh** [comment]
Seems it is not working:

```
[WARNING]: - nansencenter.django was NOT installed successfully: Failed to get
data from the API server (https://galaxy.ansible.com/api/): Failed to validate
the SSL certificate for galaxy.ansible.com:443. Make sure your managed systems
have a valid CA certificate installed. If the website serving the url uses SNI
you need python >= 2.7.9 on your managed machine  (the python executable used
(/usr/bin/python) is version: 2.7.6 (default, Oct 26 2016, 20:30:19) [GCC
4.8.4]) or you can install the `urllib3`, `pyOpenSSL`, `ndg-httpsclient`, and
`pyasn1` python modules to perform SNI verification in python >= 2.6. You can
use validate_certs=False if you do not need to confirm the servers identity but
this is unsafe and not recommended. Paths checked for this platform:
/etc/ssl/certs, /etc/pki/ca-trust/extracted/pem, /etc/pki/tls/certs, /usr/share
/ca-certificates/cacert.org, /etc/ansible. The exception msg was: hostname
u'galaxy.ansible.com' doesn't match either of
'*.c1e4.galaxy.openshiftapps.com', 'c1e4.galaxy.openshiftapps.com'.
ERROR! - you can use --ignore-errors to skip failed roles and finish processing the list.
```

**Comment by akorosov** [comment]
@mortenwh , can you check now? I was able to reproduce and fix the bug.

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+
+  config.vm.box = "ubuntu/trusty64"
```
Comment (korvinos) [inline Vagrantfile]: Suggestion. `Ubuntu/trusty64` is a little outdated. I have not tried 18.04 yet, but 16.04 LTE works fine with vagrant. Thus, shall we gradually move on a new version of the OS?
`config.vm.box = "ubuntu/xenial64"
`

Comment (akorosov) [reply Vagrantfile]: I have tested with provisioning of trusty and xenial VMs and discovered that it takes slightly longer time for xenial (although the box is slightly smaller).  Also there are 10 times more downloads of Trusty at Hashicorp. I believe that more users means more developers and more stable releases. Trusty has LTS until April 2019. Let's see then...

**Pull request merged**